pkg/ui/src/views/cluster/containers/nodesOverview/index.tsx

```diff
@@ -112,22 +114,47 @@ class LiveNodeList extends React.Component<NodeCategoryListProps, {}> {
               },
               sort: (ns) => ns.started_at,
             },
-            // Used Capacity - displays the total persisted bytes maintained by the node.
-            {
-              title: "Used Capacity",
-              cell: (ns) => Bytes(BytesUsed(ns)),
-              sort: (ns) => BytesUsed(ns),
-            },
             // Replicas - displays the total number of replicas on the node.
             {
               title: "Replicas",
               cell: (ns) => ns.metrics[MetricConstants.replicas].toString(),
               sort: (ns) => ns.metrics[MetricConstants.replicas],
             },
+            // CPUs - the number of CPUs on this node
+            {
+              title: "CPUs",
+              cell: (ns) => ns.num_cpus,
```

Comment:
it seems weird to sort on num CPUs, but it's also a little weird that this column wouldn't be sortable...